### ISSUE TITLE
Protect sensitive information

### ISSUE THREAD

**Issue by ylavoie** [lumoslabs/txgh#41]
The configuration url can display sensible information on error. Protect or filter out.

**Comment by camertron** [comment]
Hey @ylavoie, I think you mean _sensitive_ information. The configuration URL is meant to be passed in as an environment variable, and in the `raw` case is really only meant for docker containers. Txgh reads this config on initialization, so none of the information can be filtered out. Can you expand a bit on your thoughts here?

**Comment by ylavoie** [comment]
Yes I meaned sensitive information; sorry french word use in an English phrase. I had an error and all the variables were shown on the end user screen because the package is published by default in development mode.

**Comment by camertron** [comment]
Ah ok interesting. Were you using the `raw://` config scheme? I'm still not sure I understand how the config was leaked to the end user.

**Comment by ylavoie** [comment]
The Dockerfile is set for development instead of production
